services/cron/tasks.go

```diff
@@ -121,6 +122,12 @@ func GetTask(name string) *Task {
 // RegisterTask allows a task to be registered with the cron service
 func RegisterTask(name string, config Config, fun func(context.Context, *user_model.User, Config) error) error {
 	log.Debug("Registering task: %s", name)
+
+	i18nKey := "admin.dashboard." + name
+	if _, ok := translation.TryTr("en-US", i18nKey); !ok {
```

Comment:
Why a missing translation will stop the task running? We just need to record the error but the task could run continuely?

Reply:
It would only occur development and force developers to add the translation for `en-US`.

It won't affect normal users.

Reply:
I can't find any condition to run it only in dev mode.

Reply:
It's impossible to trigger such error on user side. If there is something wrong with the translation, it won't pass CI and won't be release.

So, users can always get correct releases without such error.

If a user meet such error, then their `locale_en-US.ini` must be broken and they must repair it, it's still reasonable to do `fatal error` here.